src/VaultSharp/VaultSharp.csproj

```diff
@@ -35,6 +35,12 @@
     <WarningLevel>4</WarningLevel>
     <DocumentationFile>bin\Release\VaultSharp.XML</DocumentationFile>
   </PropertyGroup>
+  <PropertyGroup>
+    <SignAssembly>true</SignAssembly>
+  </PropertyGroup>
+  <PropertyGroup>
+    <AssemblyOriginatorKeyFile>VaultSharp.snk</AssemblyOriginatorKeyFile>
```

Comment:
Moved the snk file to the keys folder in fe249a9ffd72befbcb6c7b480eb9fbca30c39725.

While you are welcome to replace the snk file, there is no point. The requirement, at least from my team, is the VaultSharp assemblies downloaded via NuGet are strong named. The actual snk file is irrelevant. As discussed previously, customers using VaultSharp can compile it from source, strong naming it with their own snk file, if required.

Similarly, digitally signing the assembly (e.g. using `signtool.exe` with a pfx file) is not required. While I have not specifically looked, I am not aware of any open source projects that do so, particularly when you need to purchase a certificate and private key. Once again. customers can procure their own certificate and private key if a digital signature is required.